src/js/load-package.ts

```diff
@@ -94,15 +94,27 @@ type PackageLoadMetadata = {
   depends: string[];
   done: ResolvablePromise;
   installPromise?: Promise<void>;
+  packageData?: PackageData;
 };
 
+export type PackageType =
+  | "package"
+  | "cpython_module"
+  | "shared_library"
+  | "static_library";
+
 // Package data inside pyodide-lock.json
 export type PackageData = {
+  name: string;
+  version: string;
   file_name: string;
-  shared_library: boolean;
-  depends: string[];
-  imports: string[];
   install_dir: string;
+  sha256: string;
+  package_type: PackageType;
+  imports: string[];
+  depends: string[];
+  unvendored_tests: boolean;
+  shared_library: boolean; // This field is deprecated
```

Comment:
Can we just get rid of `shared_library`? It isn't used internally and it's better to clean things up before exposing them.

Reply:
Maybe we can leave this to a followup...

Reply:
I think we can get rid of it.